messages/jsonschema/scripts/templates/ruby.rb.erb

```diff
@@ -24,6 +24,29 @@ module Cucumber
       attr_reader :<%= underscore(property_name) %>
     <%- end -%>
 
+      def self.from_json(json)
```

Comment:
One thing I liked about Aurellien's iteration was that he'd included RDoc with examples of calls to the API. I know we have RSpec examples but I'd like to see some docs on these methods too.

Reply:
Yeah put them in. if you want to. I was focussed on making the tests pass :-)